conans/test/functional/layout/test_editables_layout.py

```diff
@@ -17,7 +17,10 @@ def layout(self):
         self.cpp.build.includedirs = ["my_include"]
         self.cpp.build.libdirs = ["my_libdir"]
         self.cpp.build.libs = ["hello"]
-        self.cpp.build.frameworkdirs = []  # Empty list is also explicit priority declaration
+
+        # TODO: check which is the desired behaviour here... an empty list will not overwrite the
+        # conanfile cpp_info, so this will not override the default value already set.
+        self.cpp.build.frameworkdirs = []
```

Comment:
I think we need a way to say, in editables I don't have/want whatever.
I agree the "clear_none" and all that stuff is too much complicated but there might be another way.